.jscsrc

```diff
@@ -0,0 +1,33 @@
+{
+    "requireCamelCaseOrUpperCaseIdentifiers": true,
+    "requireCapitalizedConstructors": true,
+    "requireCommaBeforeLineBreak": true,
+    "requireSemicolons": true,
+
+    // spaces
+    "requireSpaceBeforeBlockStatements": true,
+    "requireSpaceBeforeKeywords": [
+        "else",
+        "while",
+        "catch"
+    ],
+    "requireSpaceAfterKeywords": true,
+    "requireSpaceBeforeObjectValues": true,
+    "requireSpaceBetweenArguments": true,
+    "requireSpacesInConditionalExpression": true,
+    "requireSpacesInFunction": {
+        "beforeOpeningRoundBrace": true,
```

Comment:
Oh lol this is totally on me. I guess that was my style back when I first wrote this! - ick!

Thanks for going along with it, but I think I'd prefer if we skipped the spacing. What do you think?